pkg/controller/worker/machines.go

```diff
@@ -214,6 +217,7 @@ func computeDisks(namespace string, pool extensionsv1alpha1.WorkerPool) (map[str
 			if vol.Type != nil {
 				dataDisk["category"] = *vol.Type
 			}
+			dataDisk["encrypted"] = true
```

Comment:
Please handle encrypted logic in other logic. There will be side effect when a shoot is reconciled.